docs/articles/documentation/recipes/debug-in-webstorm.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+title: Debug in WebStorm
+permalink: /documentation/recipes/debug-in-webstorm.html
+---
+# Debug in WebStorm
+
+This topic describes how to debug your TestCafe tests in [WebStorm](https://www.jetbrains.com/webstorm/) with a Node.js debugger.
+
+## Step 1 - Install TestCafe
+
+Execute the following command:
+
+```sh
+npm --save-dev testcafe
+```
+
+It installs TestCafe locally and adds it to the `devDependencies` section in `package.json`.
+
+```json
+{
+    "devDependencies": {
+        "testcafe": "*"
+    }
+}
+```
+
+## Step 2 - Add a Command To Debug TestCafe
+
+Add a `debug` script to the `scripts` section in `package.json`. This script contains a command that runs TestCafe and specifies the target browser.
+
+```json
+{
+    "scripts": {
+        "debug": "testcafe $NODE_DEBUG_OPTION chrome"
+    },
+
+    "devDependencies": {
+        "testcafe": "*"
+    }
+}
+```
+
+The `$NODE_DEBUG_OPTION` placeholder marks a place where Node.js debugger options are inserted before execution.
+
+Path to test files will be specified in the debug configuration at the next steps.
```

Comment:
will be -> are. We are not discussing something that is going to happen in the future, this is already described, so there is no need to use future tense here.

Reply:
'at the next steps' Why 'at'?